draft-irtf-cfrg-opaque.md

```diff
@@ -890,20 +889,20 @@ Input:
 - password, an opaque byte string containing the client's password.
 - blind, an OPRF scalar value.
 - response, a RegistrationResponse structure.
-- server_identity, the optional encoded server identity.
-- client_identity, the encoded client identity.
+- server_identity, the optional server identity.
+- client_identity, the optional client identity.
```

Comment:
```suggestion
- client_identity, the optional client identity, an octet string.
```

Reply:
Does this applies to all occurrences of `*_identity`?

Reply:
When used as an octet string, yes.